rest-service/dev-requirements.txt

```diff
@@ -1,4 +1,4 @@
-git+https://github.com/cloudify-cosmo/cloudify-common@master#egg=cloudify-common==4.5.5.dev1
+git+https://github.com/cloudify-cosmo/cloudify-common@3e5f3b223d5e5acc49926262cf5f216b194d7a00#egg=cloudify-common==4.5.5.dev1
```

Comment:
you can use the branch name, and should because you want that this branch will get updated with all changes from there